release/sample-service.yaml

```diff
@@ -8,11 +8,10 @@ gitTag: true
 gitTagDestination: 'https://github.com/edgexfoundry/sample-service.git'
 dockerImages: true
 dockerSource:
-  - 'https://nexus3.edgexfoundry.org/..'
-  - 'https://nexus3.edgexfoundry.org/..'
+  - 'nexus3.edgexfoundry.org:10004/sample-service:master'
```

Comment:
@lranjbar Is this what you envisioned the docker url's would look like? I removed `https://` because it is not really needed

Reply:
Yeah we can omit the `https://` this looks like it makes more sense.

Small change I was thinking we might want to build the URL a little bit to get ready when we release off something other than `master`

```
releaseStream: 'master'
dockerSource:
  - 'nexus3.edgexfoundry.org:10004/sample-service'
```

Reply:
That looks good to me.